games-arcade/rocksndiamonds/rocksndiamonds-4.0.0.2.recipe

```diff
@@ -48,14 +48,15 @@ BUILD_PREREQUIRES="
 
 BUILD()
 {
-	make sdl RW_GAME_DIR=$settingsDir/rocksndiamonds
+	make #RW_GAME_DIR=$settingsDir/rocksndiamonds
```

Comment:
right, still looking into it, but so far no luck with USER_SETTINGS_FILES or GLOBAL_WRITABLE_FILES :/